src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -289,6 +355,16 @@ public static SnapshotMode parse(String value, String defaultValue) {
                     + "'initial' (the default) to specify the connector should always perform an initial sync when required; "
                     + "'never' to specify the connector should never perform an initial sync ");
 
+    public static final Field BIGINT_UNSIGNED_HANDLING_MODE = Field.create("bigint.unsigned.handling.mode")
+            .withDisplayName("BIGINT UNSIGNED Handling")
+            .withEnum(BigIntUnsignedHandlingMode.class, BigIntUnsignedHandlingMode.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 1))
+            .withWidth(Width.SHORT)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withDescription("Specify how BIGINT UNSIGNED columns should be represented in change events, including: "
+                    + "'string' (the default) represent values using Java's 'string'; "
+                    + "'long' represents values using Java's 'long', which may not offer the precision but will be far easier to use in consumers.");
```

Comment:
One option is missing

Reply:
Made another PR to add the description for 'precise' mode for the option description